yr/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,496 @@
+
+![Math Equality](../images/number_guessing_game/image40.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Next, click the equal sign, and in the drop-down menu, pick the greater than sign (>). 
+
+![Math Inequality](../images/number_guessing_game/image39.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Let's fill in these empty values! In the first space, put in **myGuessSlot.value** from the **myGuessSlot** drawer and in the second space, put in <span class = "variables"> get global secret</span> from the **Variables** drawer. 
+
+![Insert values](../images/number_guessing_game/image50.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+We've now set our first if statement! **IF** our **myGuessSlot** value is greater than our secret value, then the expression within the first **then** statement will be run. Let's define this first **then** statement - we want Alexa to **ask**: "**{myGuessSlot}** is too big. Guess again". To do this, we need to once again use the **ask** block from the **Voice** drawer, the **myGuessSlot.value** block, and the <span class = "text">join</span> block from the **Text** drawer. Try it yourself! (Hint: How many parts are there in Alexa's response? Do we need to add parts to the **join** block? Click the hint to check your blocks!)
+
+<hint markdown = "block" title = "Answer!">
+If done correctly, your blocks should look like this: 
+
+![Answer!](../images/number_guessing_game/image30.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+(Don’t forget the spaces! Additional note: in order to make the game a little bit more intense, in our example we added a pause before the Intent gives the answer. Feel free to to use adjust the pause time based on your preference, you can find the pause function in the **Voice** drawer)
+</hint>
+
+<p style="padding-bottom: 7px"></p>
+
+## Define the Intent Handler for *makeGuessIntent* 
+
+We've now programmed the **makeGuessIntent** such that **if {myGuessSlot}** is greater than our **secret** value, we tell the user information about **{myGuessSlot}** and prompt the user to guess again. But what if **{myGuessSlot}** is smaller than our **secret** value? We need to define the **else if** and **then** statement to have Alexa know what to do in this situation.
+
+In order to define the **else if** statement, follow the same steps as before for defining the **if statement**, using the <span class = "math">equality</span> function block to compare **{myGuessSlot}** and our **secret** value - but this time change the greater than sign to a less than sign (<).
+
+Now let's define the second **then** statement. Follow the same steps as the first **then** statement, but instead of having Alexa **ask**: "{myGuessSlot} is too **big**. Guess again", we want Alexa to **ask**: "{myGuessSlot} is too **small**. Guess again". 
```

Comment:
Previous page you had {myGuessSlot} bolded.  Be consistent throughout.

Reply:
Fixed